cogs/edit_message.py

```diff
@@ -122,7 +122,7 @@ async def edit_message(self, ctx: TeXBotApplicationContext, str_channel_id: str,
         except discord.NotFound:
             await self.command_send_error(
                 ctx,
-                message=f"Message with ID \"{message_id}\" does not exist.",
+                message=f'Message with ID "{message_id}" does not exist.',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes